docs/vectorizer-api-reference.md

```diff
@@ -1014,13 +1014,21 @@ Return the number of pending items for the vectorizer with ID 1:
   SELECT ai.vectorizer_queue_pending(1);
   ```
 
+Use the optional `exact_count` parameter to control whether you get an exact count
+or a count capped at 1000.
+
+  ```sql
+  SELECT ai.vectorizer_queue_pending(1, exact_count=>true);
+  ```
+
 #### Parameters
 
 `ai.vectorizer_queue_pending function` takes the following parameters:
 
 |Name| Type | Default | Required | Description |
 |-|------|-|-|-|
 |vectorizer_id| int  | -|✔|The identifier of the vectorizer you want to check|
+|exact_count| bool | false |✖|If true, return exact count. If false, cap at 1000|
```

Comment:
Nit: not sure if `cap` is a short hand for `capped` but for me, a non-native speaker, `cap` can stand for other things.

```suggestion
|exact_count| bool | false |✖|If true, return exact count. If false, capped at 1000|
```